sdk/communication/AzureCommunicationChat/Tests/TestSettings.swift

```diff
@@ -28,8 +28,22 @@ import AzureTest
 import Foundation
 
 class TestSettings: TestSettingsProtocol {
-    var endpoint: String?
-    var user1: String?
-    var user2: String?
-    var token: String?
+    var endpoint = "https://endpoint/"
+    var user1 = "user1"
+    var user2 = "user2"
+    var token: String = {
+        let fakeValue = "{\"iss\":\"ACS\",\"iat\": 1608152725,\"exp\": 1739688725,\"aud\": \"\",\"sub\": \"\"}"
```

Comment:
Just curious, what format is this?

Reply:
JSON string representation of their token object.